tests/Doctrine/Migrations/Tests/MigrationRepository/MigrationRepositoryTest.php

```diff
@@ -128,25 +96,6 @@ public function testDuplicateLoadMigrationInstance() : void
         Helper::registerMigrationInstance($this->migrationRepository, new Version('Z'), $this->createMock(AbstractMigration::class));
     }
 
-    public function testFindMigrations() : void
```

Comment:
this test should be kept, what is not important is the order of returned migrations